nevergrad/functions/irrigation/data/soil/ec3.soil

```diff
@@ -0,0 +1,55 @@
+** $Id: ec3.new 1.2 1997/09/18 17:33:54 LEM release $
```

Comment:
what's the license with the data files? This is dangereous

Reply:
https://joinup.ec.europa.eu/collection/eupl/matrix-eupl-compatible-open-source-licences suggests that it is ok: we have incorporation rights.


That said you know license stuff 1000x beter than me :)

Reply:
it's really unclear to me, from what I understand of this matrix it means that it's possible to include MIT licenses code into EUPL, but we're doing the reverse, and I can't find the information

Reply:
Workaround: download from www.